src/main/java/com/iota/iri/storage/rocksDB/RocksDBPersistenceProvider.java

```diff
@@ -598,4 +607,31 @@ private DBOptions createOptions(String logPath, String configFile) throws IOExce
 
         return options;
     }
+    
+    // https://stackoverflow.com/a/41084504/4512850 -ish
+    private static Map<String, Properties> parseINI(InputStream stream) throws IOException {
```

Comment:
I think you can make this public and move it to `IotaIOUtils`

Could be useful so why not?